### PR TITLE
Update and restructure evilNeuro.js

### PR DESCRIPTION
**Pull request by tioopental**
- added/deleted/replaced prompts
- reshuffle

### CODE REVIEW

File: src/prompts/evilNeuro.js [hunk 1]
```diff
@@ -48,20 +49,28 @@ export default {
     'random sound',
     'wholesome moment',
     '"GOOD THOUGHTS"',
-    'mentions China',
```
Comment (tioopental) [inline src/prompts/evilNeuro.js:51]: prolly one time thing

Comment (cjmaxik) [reply src/prompts/evilNeuro.js:51]: Yep

File: src/prompts/evilNeuro.js [hunk 2]
```diff
@@ -48,20 +49,28 @@ export default {
...
     // anny related
-    'denies anny being her mom',
+    'mentions anny disowning her',
     'roasts anny',
 
     // neuro related
     'loves Neuro',
     'hates Neuro',
     'calls Neuro a human',
-    'asks who Neuro is',
```
Comment (tioopental) [inline src/prompts/evilNeuro.js:64]: outdated

File: src/prompts/evilNeuro.js [hunk 3]
```diff
@@ -48,20 +49,28 @@ export default {
...
     'women joke',
+    'twitchy',
+    'cursed sound',
+    'actually swears',
+    
+    // mentions
+    'mentions Hiyori Momose',
+    'mentions NASA',
+    'mentions The Swarm',
+    'mentions her missing cat',
```
Comment (cjmaxik) [inline src/prompts/evilNeuro.js:62]:
```suggestion
    'her cat is missing again',
```
?

File: src/prompts/evilNeuro.js [hunk 4]
```diff
@@ -76,6 +85,9 @@ export default {
     'chat spams :SCHIZO:',
     'chat spams :Susge:',
     'chat spams :monkaW:',
+    'chat spams :Evilge:',
+    'chat spams :COPIUM:',
+    'chat spams :BASED:',
     'chat prays',
     'chat spams "That\'s evil" copypasta'
```
Comment (cjmaxik) [inline src/prompts/evilNeuro.js:92]: I don't think I saw any of that today. Prolly remove.

File: src/prompts/evilNeuro.js [hunk 5]
```diff
@@ -48,20 +49,28 @@ export default {
     'random sound',
     'wholesome moment',
     '"GOOD THOUGHTS"',
```
Comment (cjmaxik) [inline src/prompts/evilNeuro.js:51]: One time thing too

File: src/prompts/evilNeuro.js [hunk 6]
```diff
@@ -48,20 +49,28 @@ export default {
...
-    'mentions China',
-    'mentions Hiyori Momose',
     'stares',
     'women joke',
+    'twitchy',
```
Comment (cjmaxik) [inline src/prompts/evilNeuro.js:54]:
```suggestion
    '"Twitchy"',
```

File: src/prompts/evilNeuro.js [hunk 7]
```diff
@@ -48,20 +49,28 @@ export default {
...
+    'cursed sound',
+    'actually swears',
+    
+    // mentions
+    'mentions Hiyori Momose',
+    'mentions NASA',
+    'mentions The Swarm',
+    'mentions her missing cat',
+    'mentions Goody Two Shoes',
+    'mentions her experiments on humans',
```
Comment (cjmaxik) [inline src/prompts/evilNeuro.js:64]: Too long

**Review by cjmaxik — CHANGES_REQUESTED** [review]

File: src/prompts/evilNeuro.js (hunk 3 above)
Comment (tioopental) [reply src/prompts/evilNeuro.js:62]: ye looks better

**Review by cjmaxik — APPROVED** [review]

**Pull request merged**